precompiles/relay-encoder/RelayEncoder.sol

```diff
@@ -83,64 +77,59 @@ interface RelayEncoder {
     /// @custom:selector 414be337
     /// @param rewardDestination: the account that should receive the reward
     /// @return result The bytes associated with the encoded call
-    function encodeSetPayee(bytes memory rewardDestination)
-        external
-        pure
-        returns (bytes memory result);
+    function encodeSetPayee(
+        bytes memory rewardDestination
```

Comment:
this would better be bytes32 I think as well, what do you think? now that we are breaking the API..

Reply:
We could update it, but I feel that having it static size makes it harder to manually test it. We would also need to change `encodeBond` to match `rewardDestination` as `bytes32`. If we decide to do it, maybe we could in another PR